docker-compose/docker-compose.yaml

```diff
@@ -48,7 +48,7 @@ services:
       - /srv/buildbot/helper_files:/srv/buildbot/helper_files:ro
       - ./logs/nginx:/var/log/nginx
       - ./certbot/www/:/var/www/certbot/:ro
-      - ./certbot/conf/:/etc/nginx/ssl/:ro
+      - ./certbot/ssl/:/etc/nginx/ssl/:ro
```

Comment:
If host certbot will be used why not use standard paths?
1. mounting /etc/letencrypt/live on /etc/nginx/ssl
2. use a standard path for www like /use/share/

This way we can avoid a condition in the post-hook to distinguish between /srv/dev/... (DEV) and /srv/buildbot/master/... (PROD)

Reply:
Good comment, but I like that the process of certificate renewal and the container is as much isolated as possible. That is, nginx should not have access to something in /etc/. It's probably a bit overkill but does not cost a lot and come with an extra security.

Reply:
Sure!